lib/v2/screens/send_scanner/interactor/mappers/scanner_state_mapper.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:seeds/v2/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/v2/screens/send_scanner/interactor/viewmodels/send_scanner_state.dart';
+
+class ScannerStateMapper extends StateMapper {
```

Comment:
Will this mapper be used in a near future? I do not see it is implemented anywhere.

Reply:
not here, it ended up being very simple. Removed

Reply:
no removed